artifactory/commands/transferfiles/filesdiff.go

```diff
@@ -124,6 +130,15 @@ func (f *filesDiffPhase) handleDiffTimeFrames() error {
 	// Done channel notifies the polling go routines that no more tasks are expected.
 	doneChan := make(chan bool, 2)
 
+	errorChannel := make(chan FileUploadStatusResponse, errorChannelSize)
+	go func() {
+		err := WriteTransferErrorsToFile(f.repoKey, f.getPhaseId(), convertTimeToEpochMilliseconds(f.startTime), errorChannel)
+		if err != nil {
+			// TODO: check what to do with the error
+			log.Error(err)
```

Comment:
Instead of this logging, the error should be saved, and be checked right after -
```
runWaitGroup.Wait()
```